api_app/filters.py

```diff
@@ -22,6 +22,9 @@ class JobFilter(filters.FilterSet):
     file_name = filters.CharFilter(lookup_expr="icontains")
     file_mimetype = filters.CharFilter(lookup_expr="icontains")
     tags = filters.BaseInFilter(field_name="tags__label", lookup_expr="in")
+    playbook_to_execute = filters.CharFilter(
+        field_name="playbook_to_execute__name", lookup_expr="in"
+    )
```

Comment:
I think this doesn't work:

* If i try to add a filter from the job table (es: "Dns") the url correctly change:
```http://localhost:3001/jobs?ordering=-received_request_time&playbook_to_execute=Dns```
And the backend log is:
```
2023-10-16 08:46:24,460 - api_app.views - get_queryset - INFO - user: a request the jobs with params: <QueryDict: {'page_size': ['10'], 'page': ['1'], 'received_request_time__gte': ['2023-10-09T08:00:00.000Z'], 'ordering': ['-received_request_time'], 'playbook_to_execute': ['Dns']}>
```
but in the network tab there are two requests for:
```http://localhost:3001/api/jobs?page_size=10&page=1&received_request_time__gte=2023-10-09T08:00:00.000Z&ordering=-received_request_time&playbook_to_execute=Dns```
One is an error and the other contains the response:
```{"count":0,"total_pages":1,"results":[]}```